js/apps/account-ui/src/account-security/SigningIn.tsx

```diff
@@ -1,20 +1,22 @@
 import {
-  Button,
-  DataList,
-  DataListAction,
-  DataListCell,
-  DataListItem,
-  DataListItemCells,
-  DataListItemRow,
-  Dropdown,
-  DropdownItem,
-  KebabToggle,
-  PageSection,
-  Spinner,
-  Split,
-  SplitItem,
-  Title,
-} from "@patternfly/react-core";
+	Button,
+	DataList,
+	DataListAction,
+	DataListCell,
+	DataListItem,
+	DataListItemCells,
+	DataListItemRow,
+	PageSection,
+	Spinner,
+	Split,
+	SplitItem,
+	Title
+} from '@patternfly/react-core';
+import {
+	Dropdown,
+	DropdownItem,
+	KebabToggle
+} from '@patternfly/react-core/deprecated';
```

Comment:
I've [logged an issue](https://github.com/keycloak/keycloak/issues/28197) to clean up these deprecated components later.